src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -218,12 +218,19 @@
   <PropertyGroup>
     <GetNuGetPackageVersionsDependsOn>$(GetNuGetPackageVersionsDependsOn);CreateVersionInfoFile</GetNuGetPackageVersionsDependsOn>
   </PropertyGroup>
+    
+  <PropertyGroup>
+    <SyncInfoDirectory Condition="'$(SyncInfoDirectory)' == ''">$(ObjDir)</SyncInfoDirectory>
+    <SyncInfoFile Condition="'$(SyncInfoFile)' == ''">$(SyncInfoDirectory)version.txt</SyncInfoFile>
```

Comment:
I considered this, but for official builds, where we are shipping packages from, this will not be an issue.